image/conf/config.yml

```diff
@@ -46,7 +46,62 @@ platforms :
           calibrationsAllowed: true
 
 moneytransfer:
+    # Minimum amount of money in US cent a worker has to earn before he gets paid off
     payOffThreshold: 0
+
+    # Password used to verify a valid Amazon GiftCode e-mail
+    # enter password as message on the Amazon homepage when buying GiftCodes
     parsingPassword: test
+
+    # E-mail address to send notification mails to
     notificationMailAddress: pseipd@gmail.com
-    scheduleInterval: 7
\ No newline at end of file
+
+    # Interval in days, when the workers get paid off
+    scheduleInterval: 7
+
+# Defined platforms to run
+platforms:
+    # Type to use:
+    # - mturk
+    # - pybossa
+    # - dummy
+    - type: dummy
+
+      # Standard credentials
+      user: samplePlatformUser
+      password: samplePassword
+
+      # Name which is used to display
+      name: Sandbox
+
+      # URL to the platform
+      url: http://example.com/
+
+    - type: pybossa
+
+      # Key to access the PyBossa API
+      apiKey: 8ec92fa1-1bd1-42ad-8524-3d2bab4588b1
```

Comment:
i have a docker-postures container on the docker-hub which contains a partly initialised pybossa (the user and the project was added, but not published, because the tasks have to be added by the os first). It is intended for development and therefore can be shared